testsuite/features/secondary/min_action_chain.feature

```diff
@@ -92,6 +92,16 @@ Feature: Action chains on Salt minions
     Then I should see a "Revision 1 of /etc/action-chain.cnf from channel Action Chain Channel" text
     And I should see a "Update Configuration File" button
 
+  Scenario: Download the configuration file from configuration channel
+    Given I am authorized as "admin" with password "admin"
+    When I follow the left menu "Configuration > Channels"
+    And I follow "Action Chain Channel"
+    And I follow "List/Remove Files"
+    And I follow "/etc/action-chain.cnf"
+    And I follow "Download File"
+    And I wait until file "${PWD}/action-chain.cnf" exists on "localhost"
```

Comment:
I think `${HOME}` is more correct here.. I'll test it and update PR accordingly if works.

Reply:
I've not changed location from `${PWD}` to `${HOME}`.